ouroboros-consensus-cardano/src/ouroboros-consensus-cardano/Ouroboros/Consensus/Cardano/Node.hs

```diff
@@ -56,8 +57,9 @@ module Ouroboros.Consensus.Cardano.Node (
 
 import           Cardano.Binary (DecoderError (..), enforceSize)
 import           Cardano.Chain.Slotting (EpochSlots)
+import qualified Cardano.Ledger.Api.Era as L
```

Comment:
I wonder if it'd make sense to import this `as Ledger`. Maybe `L` is clear enough, although I do think `Ledger.LatestKnownEra` reads a bit better than `L.LatestKnownEra`.

Reply:
Yeah, our qualified names for ledger imports are quite ad-hoc; we even have a ticket for that: https://github.com/input-output-hk/ouroboros-network/issues/3105